recipe/meta.yaml

```diff
@@ -1,54 +1,42 @@
+{% set name = "scikit-bio" %}
 {% set version = "0.5.2" %}
+{% set sha256 = "566c83249d2d338104d106c3a8ba36fa1afeabf10ead8e95b84605b71c65b743" %}
+{% set build_num = "0" %}
 
 package:
-  name: scikit-bio
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: scikit-bio-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/s/scikit-bio/scikit-bio-{{ version }}.tar.gz
-    sha256: 566c83249d2d338104d106c3a8ba36fa1afeabf10ead8e95b84605b71c65b743
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win]
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: true  # [win or py2k]
+  number: {{ build_num }}
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
-    - setuptools
-    - bz2file >=0.98
-    - lockfile >=0.10.2
-    - cachecontrol >=0.11.5
-    - contextlib2 >=0.4.0
-    - decorator >=3.4.2
-    - future >=0.14.3
-    - ipython >=3.2.0
-    - matplotlib >=1.4.3
-    - natsort >=4.0.3
-    - numpy >=1.9.2
-    - pandas >= 0.19.2
-    - scipy >=0.15.1
-    - six >=1.9.0
-    - nose >=1.3.7
-
+    - pip
+    - cython
+    - numpy 1.9.*
+    - nose
   run:
     - python
-    - bz2file >=0.98
     - lockfile >=0.10.2
     - cachecontrol >=0.11.5
-    - contextlib2 >=0.4.0
     - decorator >=3.4.2
-    - future >=0.14.3
     - ipython >=3.2.0
     - matplotlib >=1.4.3
     - natsort >=4.0.3
-    - numpy >=1.9.2
-    - pandas >= 0.19.2
+    - numpy >=1.9.2,<1.14.0
```

Comment:
Why does `numpy` have an upper bound on the version constraint?

Reply:
not sure. It is from `setup.py`. @ebolyen , do you know why?

Reply:
IIRC that had something to do with a docstring change, it just makes the doctests fail (that should get fixed ideally), it's not an API limitation as far as I know.